app/views/components/AboutTheGoods.scala.html

```diff
@@ -17,35 +17,40 @@
 @import components._
 @import viewmodels.checkAnswers.summary._
 @import viewmodels.ApplicationViewModel
+@import models.requests.Attachment
+@import models.requests.Privacy._
 
 @this(
-    layout: templates.Layout,
-    govukSummaryList: GovukSummaryList,
-    applicationSummaryList: ApplicationSummaryList,
-    heading: Heading,
-    headingH3: headingH3,
-    paragraph: Paragraph
+layout: templates.Layout,
+govukSummaryList: GovukSummaryList,
+applicationSummaryList: ApplicationSummaryList,
+heading: Heading,
+headingH3: headingH3,
+paragraph: Paragraph,
+govukTable: GovukTable,
```

Comment:
I don't think it's used here either

Reply:
Not anymore, i had an iteration of this that used a table and looked terrible 🧹